foxpuppet/windows/browser/window.py

```diff
@@ -24,6 +25,14 @@ class BrowserWindow(BaseWindow):
     _tab_browser_locator = (By.ID, 'tabbrowser-tabs')
 
     @property
+    def navbar(self):
+        """Provides access to the navbar.
+        :returns: :py:class:`~foxpuppet.windows.browser.navbar.NavBar`
+        :return type: object
+        """
+        return NavBar(self.selenium)
```

Comment:
We need to pass the window and root element, unless we implement use of `_root_locator` like we have in PyPOM.